core/asset/asset.go

```diff
@@ -17,6 +17,11 @@ const (
 	Token    AssetType = 0x11
 )
 
+const (
+	MAX_PRECISION = 8
+	MIN_PRECISION = 0
```

Comment:
forgive my captious, the underline _ is a good symbol to mark const, but in the golang, especially in our currently coding style, all variable be named with  a-z A-Z  only, the _ is not be encouraged using to keep the style consistent